man/chpl.txt

```diff
@@ -195,6 +195,12 @@ OPTIONS
   --[no-]bounds-checks   Enable [disable] run-time bounds checking,
                     e.g. during slicing and array indexing.
 
+  --[no-]formal-domain-checks   Enable [disable] run-time checks to ensure
```

Comment:
Oh, also isn't there a test that has to be updated because the --help output changed?